config/karma.saucelabs.js

```diff
@@ -210,7 +210,7 @@ module.exports = function (config) {
 
     port: 9876,
 
-    retryLimit: 3,
+    retryLimit: 0,
```

Comment:
FYI this file is ignored, we no longer run saucelabs tests.

Reply:
Removed